### ISSUE TITLE
Plugin doesn't work when BukkitValues doesn't exist

### ISSUE THREAD

**Issue by TJ20201** [TJ20201/TJsRPGPlugin#1]
When BukkitValues doesn't exist within the player (for example, in a MineHut Server), the plugin will not work at all. This can be avoided by storing player information in a separate YAML file, called something such as `players.yml`

A mock `players.yml` file, using values possible in release `v1.0.0` in a local server where BukkitValues exists:
```yaml
01234567-89ab-cdef-ghij-klmnopqrstuv:
  mana: 25
  maxMana: 40
  level: 1
  curEXP: 92
  totEXP: 115
76543210-98ba-fedc-jigh-nopqrsvutklm:
  mana: 50
  maxMana: 55
  level: 2
  curEXP: 15
  totEXP: 130
```
With just two low-level players, the file enters 201 bytes and is more customisable than the current system of data saving. Migration to this system, if implemented, will be easy using the already existing `setPlayerData` and `getPlayerData` however the creation of the YAML file will be more difficult.

**Comment by TJ20201** [comment]
This is being worked on in `v1.1.0`, and this Issue will be closed once this part of the new version is made.

**Comment by TJ20201** [comment]
Fixed in Version 1.1.0 commit 6f586a6.
